src/Components/Modal/WinModal.js

```diff
@@ -0,0 +1,45 @@
+//JS file for modal that displays when game ends
+import React, { Component } from 'react';
+import Swal from "sweetalert2"
+import Party_Face from "../Party Face.svg"
+
+//Modal that displays when game is won
+class WinModal extends Component {
+
+    constructor(props) {
+        super(props);
+
+        //props sent over are winner and the state of the game board at time of win
+        this.state = {
+            board: this.props.win_board,
+            winner: this.props.winner
+        }
+    }
+
+    render() {
+        /*var i
```

Comment:
For the next sprint, please remove the commented out unused code. Explained by @naomi-albert that this will be functional on the next sprint.